classifier_and_summarizer/summarizer/summarizer_output.py

```diff
@@ -0,0 +1,41 @@
+''' This script defines the object structure
+for the output(s) of the summarizer and provides
+methods to access the data as required'''
+
+
+class StampPages:
+    def __init__(self):
+        self.stamp_pages = list()
+
+    def add_stamp_page(self, stamp_page):
+        self.stamp_pages.append(stamp_page)
+
+    def get_formatted_list(self):
+        formatted_list = list()
+        for stamp_page in self.stamp_pages:
+            formatted_list.append(stamp_page.get_formatted_dict())
+        return formatted_list
+
+
+class StampPage:
+    def __init__(
+            self,
+            media_index,
+            is_embedded_content,
+            overlay_title,
+            overlay_text,
+            overlay_font_style,
+            overlay_font_size,
+            stamp_position):
+        self.media_index = media_index
+        self.is_embedded_content = is_embedded_content
+        self.overlay_title = overlay_title
+        self.overlay_text = overlay_text
+        self.overlay_font_style = overlay_font_style
+        self.overlay_font_size = overlay_font_size
+        self.stamp_position = stamp_position
```

Comment:
media_index = the index of the media ( images/videos/insta post etc ) as given in the contents . The media index is passed here rather than the media itself so the generator can access and modify the other attributes of the media directly from the contents list itself .

is_embedded_content = True/False - indicates if the media is an embedded content . If the content is of an embedded format we can safely ignore overlay text attributes

overlay_title and overlay_text = the text that will be displayed on top of the stamp page 

overlay_font_style and overlay_font_size = taken as is from the attributes defined for the overlay title and/or text

stamp_position = the position of the stamp page in the final order of stamps . Refer to design doc on how this is calculated